build/docker/Dockerfile

```diff
@@ -66,6 +67,7 @@ RUN wget https://artifacts.crowdin.com/repo/deb/crowdin.deb -O crowdin.deb
 RUN dpkg -i crowdin.deb
 
 # Install PIP and PDoc
+RUN wget https://bootstrap.pypa.io/2.7/get-pip.py && python get-pip.py
```

Comment:
what about adding '&& rm get-pip.py' in order to not leave the file in the image ?

Reply:
Sounds good.